src/main/java/com/hm/achievement/command/AbstractRankingCommand.java

```diff
@@ -121,6 +115,42 @@ public void executeCommand(CommandSender sender, String[] args) {
 		}
 	}
 
+	private int getPage(String[] args) {
```

Comment:
This method is basically using exceptions as control flow, which should be avoided when possible. I suggest squashing things down to:
````
private int getPage(String[] args) {
    return args.length > 1 && NumberUtils.isDigits(args[1]) ? Integer.parseInt(args[1]) : 1;
}
````
Note that index 0 in args is actually the command (for instance `top`, `month`, `week`), so we should look at index 1 instead.